src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -151,12 +149,10 @@ internal sealed partial class ManagedWebSocket : WebSocket
         /// </summary>
         private object ReceiveAsyncLock => _utf8TextState; // some object, as we're simply lock'ing on it
 
-        /// <summary>Initializes the websocket.</summary>
-        /// <param name="stream">The connected Stream.</param>
-        /// <param name="isServer">true if this is the server-side of the connection; false if this is the client-side of the connection.</param>
-        /// <param name="subprotocol">The agreed upon subprotocol for the connection.</param>
-        /// <param name="keepAliveInterval">The interval to use for keep-alive pings.</param>
-        private ManagedWebSocket(Stream stream, bool isServer, string? subprotocol, TimeSpan keepAliveInterval)
+        private readonly WebSocketInflater? _inflater;
```

Comment:
Should we combine the Inflater and Deflater into a single class and have it be a partial class so the logic is separated in source?

Reply:
It's doable, and I would have to rename some of the fields to make sure one class doesn't incidentally use the state of the other. If you think it's justified, I will do it.